tools/cli/go-whisk-cli/commands/activation.go

```diff
@@ -254,12 +255,17 @@ func lastFlag(args []string) ([]string, error) {
     }
 
     if len(activations) == 0 {
-      whiskErr := whisk.MakeWskError(errors.New("No activations found"), whisk.EXITCODE_ERR_GENERAL, whisk.DISPLAY_MSG, whisk.DISPLAY_USAGE)
+      errStr := fmt.Sprintf("%s%s %s %s", "clinet.Activation.",funcName,"error:", "No activations found")
```

Comment:
Would something like "No activations found. Be sure wsk activation list contains activations" be more clear? This is simply checking to see if they have any activations and if not alerting the user that they don't.